src/Makevars

```diff
@@ -2,7 +2,7 @@ CXX_STD = CXX11
 
 PKG_CPPFLAGS = -I../inst/include/
 PKG_CXXFLAGS = $(SHLIB_OPENMP_CXXFLAGS) -DNO_MANUAL_VECTORIZATION -DSTRICT_R_HEADERS
-PKG_LIBS = $(SHLIB_OPENMP_CXXFLAGS) $(BLAS_LIBS)
```

Comment:
Why removing BLAS is needed? despite we don't use BLAS routines in this pkg, I think it still should compile fine.

Reply:
This raises a warning
```
> checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ... WARNING
    apparently using $(BLAS_LIBS) without following $(FLIBS) in 'src/Makevars'
```

Reply:
alright, removing `$(BLAS_LIBS)`